static/templates/settings/playground_settings_admin.hbs

```diff
@@ -5,6 +5,43 @@
             Configure external sites that can run code present in a Zulip code block.
             {{/tr}}
         </p>
+
+        {{#if is_admin}}
+        <form class="form-horizontal admin-playground-form">
+            <div class="add-new-playground-box grey-box">
+                <div class="new-playground-form wrapper">
+                    <div class="settings-section-title">{{t "Add a new playground" }}</div>
+                    <div class="alert" id="admin-playground-status"></div>
+                    <div class="control-group">
+                        {{!-- TODO: This should have a typeahead suggestion popover --}}
+                        <label for="playground_pygments_language" class="control-label"> {{t "Language" }}
+                            <i class="fa fa-info-circle settings-info-icon tippy-zulip-tooltip"
+                              aria-hidden="true" data-tippy-placement="right" data-tippy-content="{{t 'The human-readable Pygments language name for the programming language' }}"></i>
```

Comment:
- Yeah, if we indeed explain the playgrounds feature in more details (as you have in the previous comment), then I don't see a need for the `i` buttons.
- The typeahead for the code blocks is a list of pygments alias. In order for us to show playgrounds per programming language, we map these playgrounds into a user-friendly canonical name (E.G: py2 and python2 point to the "Python 2.x"). 
- I'll see if I can tweak the language selector typeahead based on your suggestions :)

Reply:
@alya I don't see any auto-complete in my setup. Is this a Zulip auto-complete or from some third-party software?

Reply:
This is auto-complete form my browser. You can try to replicate by, e.g., creating a Python playground, and then going to create a second one and starting to type "p" for the language.

Reply:
> The typeahead for the code blocks is a list of pygments alias. In order for us to show playgrounds per programming language, we map these aliases into a user-friendly canonical name (E.G: py2 and python2 point to the "Python 2.x"). Hence in both the backend and frontend, we can configure only one playground per friendly name.

Thinking more about it, I'm worried that the mismatch between the code blocks and the playground menu will be confusing. If what you type in the compose box doesn't match the playground menu, how do you know whether it will trigger the playground or not?

Maybe we can put all the pygment aliases into a parenthetical? E.g. "Python 2.x (py2, python2)"?

@timabbott  what do you think?

Reply:
Yeah, I think that'd be a nice touch if we can do it.

Reply:
(You can disable the browser autocomplete with `autocomplete="off"` -- we do that in many settings inputs for this reason.